makeybutton/button.go

```diff
@@ -49,7 +54,7 @@ func NewButton(pin machine.Pin) *Button {
 
 // Configure configures the Makey Button pin to have the correct settings to detect touches.
 func (b *Button) Configure() error {
-	// Note that we have to first turn on the pullup, and then turn off the pullup,
+	// Note that on AVR we have to first turn on the pullup, and then turn off the pullup,
```

Comment:
Same holds for non-AVR? Or this driver for AVR only?

Reply:
It is only "required" for AVR, but does not do anything bad on the other platforms I tested (mostly rp2040).